<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_tools</class>
 <widget class="QDialog" name="admin_tools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Функционал администратора</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Вставка</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>291</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Количество частей</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>291</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Статус выхода</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>291</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Перевод</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>311</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>311</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Жанр</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>311</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Автор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>311</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Издатель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>400</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название комикса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Удаление</string>
    </attribute>
    <widget class="QTableWidget" name="refreshTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>271</width>
       <height>321</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Название</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>300</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить комикс</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите индекс комикса</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_line">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>211</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновление таблицы</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Поль-ли</string>
    </attribute>
    <widget class="QTableWidget" name="tableUsers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Логин</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пароль</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>190</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Вывод поль-лей</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
